README.rst

```diff
@@ -97,9 +97,13 @@ Running tests
 
 SageMaker Python SDK has unit tests and integration tests.
 
+You can install the libraries needed to run the tests by running :code:`pip install --upgrade .[test]` or, for Zsh users: :code:`pip install --upgrade .\[test\]`
+
 **Unit tests**
 
-tox is a prerequisite for running unit tests so you need to make sure you have it installed. To run the unit tests:
+To run the unit tests, you'll need to have interpreters for Python 2.7 and Python 3.5 installed.
+
+To run the unit tests with tox, run:
```

Comment:
I'd add something here that tox is a convenient way to run both versions of Python in addition to the linters, etc. that have been configured in tox.ini. Otherwise, the mention of tox seems rather random as the only requirement mentioned is having Python interpreters.